CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
-## 4.84.0 (Unreleased)
+## 4.84.1 (Unreleased)
+
```

Comment:
```suggestion

NOTES:
* The `4.85.0` release backports configuration for the retention period for Cloud Storage soft delete (https://cloud.google.com/resources/storage/soft-delete-announce) so that customers who have not yet upgraded to `5.22.0`+ are able to configure the retention period of objects in their buckets. By upgrading to this version and configuring or otherwise interacting with the `google_storage_bucket.soft_delete_policy` values, you will need to upgrade directly to `5.22.0`+ from `4.85.0` when upgrading to `5.X` in the future.
```